kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -735,4 +735,13 @@ object KyuubiConf {
       .version("1.3.0")
       .booleanConf
       .createWithDefault(true)
+
+  val SESSION_NAME: OptionalConfigEntry[String] =
+    buildConf("session.name")
+      .doc("Session name is used to relate the history session which can help optimize. " +
+        "Please set this config using different value for workflows which are session level. " +
+        "This config is none means we can not find the history session.")
```

Comment:
Session name is just a session name. History is just a typical use case, the doc is overkilling to me

Reply:
how about this updating